src/lazy/binary/raw/v1_1/immutable_buffer.rs

```diff
@@ -244,7 +244,7 @@ impl<'a> BinaryBuffer<'a> {
         };
 
         if self.len() < size_in_bytes {
-            return IonResult::incomplete("reading a flex_uint value", self.offset());
+            return IonResult::incomplete("a flex_uint value", self.offset());
```

Comment:
🪧 The error text produced by `incomplete` is:
```
"ran out of input while reading {label} at offset {position}"
```
I went through and removed all of the redundant "reading a" and "parsing a" bits from the various callsites.